src/constants.ts

```diff
@@ -15,6 +20,11 @@ const invalidRevisions = [
     '0x',
 ]
 
+export const genesis = {
```

Comment:
should we add it to validRevisions or we use it directly ?

Reply:
It's used directly https://github.com/vechain/thor-e2e-tests/blob/4d987ae4c481425a7ba9a32016c93c9569ae8ab4/src/test-env.ts#L71